Test Host/PickerController.m

```diff
@@ -99,50 +115,100 @@ - (void)viewDidLoad
     countdownSelectionTextField.returnKeyType = UIReturnKeyDone;
     countdownSelectionTextField.inputView = countdownPicker;
     countdownSelectionTextField.accessibilityLabel = @"Countdown Selection";
+    
+    self.dateCalendarPicker = [[UIDatePicker alloc] init];
+    self.dateCalendarPicker.datePickerMode = UIDatePickerModeDate;
+    [self.dateCalendarPicker addTarget:self action:@selector(dateCalendarPickerChanged:) forControlEvents:UIControlEventValueChanged];
+    [self.dateCalendarPicker setLocale:[[NSLocale alloc] initWithLocaleIdentifier:@"en_US"]];
+    
+    self.datePickerCalendarTextField.placeholder = NSLocalizedString(@"Date Calendar Selection", nil);
+    self.datePickerCalendarTextField.returnKeyType = UIReturnKeyDone;
+    self.datePickerCalendarTextField.inputView = self.dateCalendarPicker;
+    self.datePickerCalendarTextField.accessibilityLabel = @"Date Calendar Selection";
```

Comment:
AFAICT - Not seeing any tests that are interacting with `Date Calendar Selection` or `Date Time  Calendar Selection`? Totally might be missing it.

Reply:
They are in the tests